docs/templates/template_reference.md

```diff
@@ -815,9 +815,26 @@ The selected value can be changed in the profile (consult the actual variable fo
 
     -   **datatype** - data type of the sysctl value, eg. `int`.
 
-    -   **sysctlval** - value of the sysctl value, eg. `'1'`. If this
+    -   **sysctlval** - value of the sysctl value. This can be either an atomic
```

Comment:
A general rant - the sysctl template shouldn't have sysctlval and sysctlvar - those two words are very similar, and the "sysctl" part is implied. It should have "name" and "values". But don't change this in the PR - if we ever decide to change argument names, that should be done separately.